smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/TypeScriptDependency.java

```diff
@@ -66,6 +66,9 @@
     AWS_SDK_FETCH_HTTP_HANDLER("dependencies", "@aws-sdk/fetch-http-handler", "^1.0.0-alpha.1", false),
     AWS_SDK_NODE_HTTP_HANDLER("dependencies", "@aws-sdk/node-http-handler", "^1.0.0-alpha.1", false),
 
+    // Conditionally added when doing hostname validation
+    AWS_VALID_HOSTNAME("dependencies", "@aws-sdk/util-valid-hostname", "^1.0.0-alpha.0", false),
```

Comment:
Does this really need to be in a standalone package?

Reply:
Removed.  Importing this function from protocol-http now.